src/models/tools/table/table-content.ts

```diff
@@ -28,6 +28,10 @@ export function canonicalizeValue(value: number | string | undefined) {
   return isFinite(num) ? num : 0;
 }
 
+export function getRowLabel(index: number, prefix: string = "p") {
```

Comment:
A few of your PRs have introduced default params like this for the future - I like that, kind of a sneaky way to introduce a constant.